[Merge-Queue] Comment on PR when failing validation

JonWBedard opened the pull request:
#### 818a5466dfb48749d64a3568b7bb9c7a221bf511
<pre>
[Merge-Queue] Comment on PR when failing validation
<a href="https://bugs.webkit.org/show_bug.cgi?id=238969">https://bugs.webkit.org/show_bug.cgi?id=238969</a>
&lt;rdar://problem/91451392 &gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(ValidateChangeLogAndReviewer.evaluateCommand): Add BlockPullRequest on failure.
(ValidateCommitMessage.evaluateCommand): Block and comment on pull request if
validation fails.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/249437@main">https://commits.webkit.org/249437@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@292605">https://svn.webkit.org/repository/webkit/trunk@292605</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -4957,20 +4961,25 @@ def evaluateCommand(self, cmd):
+        if rc == SUCCESS:
+            self.summary = 'Validated commit message'
+            log_text = self.log_observer.getStdout()
+            if '(OOPS!)' in log_text:
```
Do you want to do regex matching here for cases like OPPS like you did earlier?

JonWBedard replied:
Good idea, will do that

JonWBedard committed:
[Merge-Queue] Comment on PR when failing validation
https://bugs.webkit.org/show_bug.cgi?id=238969
<rdar://problem/91451392>

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(ValidateChangeLogAndReviewer.evaluateCommand): Add BlockPullRequest on failure.
(ValidateCommitMessage.evaluateCommand): Block and comment on pull request if
validation fails.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/249437@main
git-svn-id: https://svn.webkit.org/repository/webkit/trunk@292605 268f45cc-cd09-0410-ab3c-d52691b4dbfc
